backend/src/packages/files/file.model.ts

```diff
@@ -1,14 +1,37 @@
+import { Model, type RelationMappings } from 'objection';
+
 import {
   AbstractModel,
   DatabaseTableName,
 } from '~/libs/packages/database/database.js';
+import { composeDatabaseRelationPath } from '~/libs/packages/database/libs/helpers/compose-database-relation-path/compose-database-relation-path.helper.js';
```

Comment:
I guess import path should be
`~/libs/packages/database/libs/helpers/helpers.js`

Reply:
Thank you, such simple mistakes are often overlooked.
I can't find how to configure auto-imports to see re-exports.

- [65654a1](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/118/commits/65654a1286d0d5908f927dd8b03a652952aea9d5)